src/redux/sagas/meta/get_CCYs_id.js

```diff
@@ -0,0 +1,16 @@
+import { put } from 'redux-saga/effects'
+import Debug from 'debug'
+import axios from 'axios'
+import marketActions from '../../actions/market'
+
+const debug = Debug('hfui:rx:s:market-hfui:getting CCYs id')
+const URL = 'http://localhost:45001/v2/conf/pub:map:currency:support:zendesk'
```

Comment:
using http://localhost:45001 wont work for hosted version, use REACT_APP_UFX_PUBLIC_API_URL.
also update this env in .env-cmdrc file for staging/prod mapping